### DISCUSSION TITLE
Util

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/OOP-using-Java#25] [Q&A] [answered]
What's the syntax for Scanner class

**Comment by Md-Inam** [comment, marked as answer]
Scanner sc = new Scanner(System.in);

**Comment by Mohammedomair23** [comment]
Scanner scanner = new Scanner(System.in);

1, Initialization = new Scanner(System.in).
2. Declaration  = Scanner scanner.
3. Assignment = assigns the newly created object to the variable (scanner).

**Comment by Maneesha743527** [comment]
Scanner sc = new Scanner(System.in
